### ISSUE TITLE
Confusion about zero-shot setting on Video-Text Retrieval

### ISSUE THREAD

**Issue by overwhelmedxh** [OpenGVLab/InternVideo#89]
Thank you for your in interesting work and your shared code!
I'm very confused that whether the zero-shot performance on MSRVTT reported in [here](https://github.com/OpenGVLab/InternVideo/tree/main/Downstream/Video-Text-Retrieval#our-results) requires setting “--mergeclip=True”?
Below is the result I reproduced：
“--mergeclip=True”：
![image](https://github.com/OpenGVLab/InternVideo/assets/88419969/e8920520-9b7b-434b-9092-d82acf07d8ec)
“--mergeclip=False”：
![image](https://github.com/OpenGVLab/InternVideo/assets/88419969/be7a3083-93d9-4af6-9ef0-374ad3b9ae33)

AS the provided [file](https://github.com/OpenGVLab/InternVideo/blob/main/Downstream/Video-Text-Retrieval/zeroshot_scripts/eval_msrvtt.sh) defaults to "--mergeclip=True", I wonder if there is something wrong with this.

**Comment by 1240446371** [comment]
it seems that when setting “merge=True”,the results are better than the paper presented?

**Comment by overwhelmedxh** [comment]
> it seems that when setting “merge=True”,the results are better than the paper presented?

Yes. It seems that the results reported in the paper are obtained by setting “merge=True” without DSL.

**Comment by 1240446371** [comment]
> > it seems that when setting “merge=True”,the results are better than the paper presented?
> 
> Yes. It seems that the results reported in the paper are obtained by setting “merge=True” without DSL.

I test the performance on activityNet，and obtain better results on “merge=True” with DSL，but obtain worse results on “merge=True” without DSL（worse than paper presented）. The author replied to another people that they use DSL results. I also confuse about which setting they use ~~

**Comment by Hari-Durai-Baskar** [comment]
> > > it seems that when setting “merge=True”,the results are better than the paper presented?
> > 
> > 
> > Yes. It seems that the results reported in the paper are obtained by setting “merge=True” without DSL.
> 
> I test the performance on activityNet，and obtain better results on “merge=True” with DSL，but obtain worse results on “merge=True” without DSL（worse than paper presented）. The author replied to another people that they use DSL results. I also confuse about which setting they use ~~

Hi, were u able to resolve the confusion?
